docs/Tutorials/Kubernetes/Production.md

```diff
@@ -44,19 +39,37 @@ Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCom
 
 ### Deploy the network
 
-For the rest of this tutorial we use the [**Helm charts**](https://github.com/ConsenSys/quorum-kubernetes/tree/master/helm). After you have cloned the [Quorum-Kubernetes](https://github.com/ConsenSys/quorum-kubernetes) repository,
-change the directory to `prod` for the rest of this tutorial.
+#### Check that you can connect to the cluster with `kubectl`
```

Comment:
@joshuafernandes is this step required? It appears to duplicate line 30

Reply:
Can probably scrap it, me just being paranoid ;)